src/sentry/api/endpoints/organization_member_commits.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from sentry.api.bases import OrganizationMemberEndpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize, CommitWithReleaseSerializer
+from sentry.db.models.query import in_iexact
+from sentry.models import Commit, CommitAuthor, UserEmail
+
+
+class OrganizationMemberCommitsEndpoint(OrganizationMemberEndpoint):
+    def get(self, request, organization, member):
+        email_list = list(UserEmail.objects.filter(
+            user=member.user_id,
+            is_verified=True,
+        ).values_list('email', flat=True))
+        if email_list:
+            queryset = Commit.objects.filter(
+                organization_id=organization.id,
+                author__in=CommitAuthor.objects.filter(
+                    in_iexact('email', email_list),
+                    organization_id=organization.id,
+                )
+            ).order_by('-date_added')
```

Comment:
For this, we _don't_ have an index that'll be used here for pagination. I think we might want to add one, but we can see. The closest index we have is `"sentry_commit_repository_id_5b0d06238a42bbfc" btree (repository_id, date_added)`

So we might want to add one that covers like, `organization_id, author_id, -date_added` to make this nice.